pkg/container/container.go

```diff
@@ -72,13 +94,115 @@ type container struct {
 	id         string
 	name       string
 	privileged bool
+	spec       generate.Generator
 }
 
 // New creates a new, empty Sandbox instance
-func New(ctx context.Context) Container {
+func New(ctx context.Context) (Container, error) {
+	spec, err := generate.New("linux")
+	if err != nil {
+		return nil, err
+	}
 	return &container{
-		ctx: ctx,
+		ctx:  ctx,
+		spec: spec,
+	}, nil
+}
+
+// SpecAddMount adds a specified mount to the spec
+func (c *container) SpecAddMount(r rspec.Mount) {
```

Comment:
Now we have two docstrings for SpecAddMount. I think one is enough

Reply:
2. Perhaps `s/SpecAddMount/AddMount/`?

Reply:
there's similar comments over other functions in the file, I think I we should either remove all of them or keep all of them.